src/libcore/result.rs

```diff
@@ -729,6 +729,26 @@ impl<T, E: fmt::Debug> Result<T, E> {
                 panic!("called `Result::unwrap()` on an `Err` value: {:?}", e)
```

Comment:
This could now turn into just `self.expect("called `Result::unwrap()` on an `Err` value")`.